### PR TITLE
fix id issue

### LINKED ISSUE #4: kremling-loader and runtime css string do not work together
If you're using both raw css and postcss, both have a counter that starts at zero and increment from there. This makes the css rules generated conflict with each other.

```css
/* generated with kremling-loader */
[kremling="0"] .hello, [kremling="0"].hello {
  color: navajowhite;
}

/* generated with raw css in browser */
[kremling="0"] .hello, [kremling="0"].hello {
  color: lawngreen;
}
```

^^ This is a problem because they have the same "0" which makes their css scopes overlap. Suggested solution would be to append a prefix or something in front of kremling-loader's attribute values. Maybe `[kremling="p0"] .hello`? `P` for `postcss`?? idk

### LINKED ISSUE #5: kremling-loader's namespace option is inconsistent with Scoped's namespace option
In the Scoped component, a namespace of `hi` will turn into the following:

```css
<div data-hi="0">There</div>
```

In kremling-loader, a namespace of `hi` will turn into the following:

```css
<div hi="0">There</div>
```

In other words, Scoped prepends `data-` to the attribute name, whereas kremling-loader does not. Afaik, there are no bugs associated with this behavior, but I do think it would be nice from the user's perspective for `namespace` to have a consistent meaning regardless of whether you're using Scoped, kremling-loader, or useCss.

### PR DESCRIPTION
**Pull request by geoctrl**
- fixes #4 
- if `namespace` is undefined, then it uses the default `kremling` (changed from `data-kremling`)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 const postcssKremling = require('./src/postcss-kremling-plugin');
 
+let globalId = 0;
+
 module.exports = function(source) {
+  const id = globalId;
+  globalId += 1;
   const loaderOptions = getOptions(this) || {};
-  let kremlingNamespace = 'data-kremling';
+  let kremlingNamespace = 'kremling';
```
Comment (geoctrl) [inline index.js]: any time the `kremling` repo changes the default namespace we have to change it here too

not sure how else to handle this (besides making it required)

File: index.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
 const { getOptions } = require('loader-utils');
 const postcss = require('postcss');
-const kremId = require('./src/kremling-id');
 const postcssKremling = require('./src/postcss-kremling-plugin');
 
+let globalId = 0;
```
Comment (geoctrl) [inline index.js]: the `kremId` stuff was overcomplicated - just added a global Id that increments

Comment (joeldenning) [reply index.js]: 👍

File: index.js (hunk 1 above)
Comment (joeldenning) [reply index.js]: 👍 i think that's fine. I don't think we'll change the default namespace again, at least for the foreseeable future. The `data-` prefix is the only thing that i can think of that we would have wanted to change.

**Pull request merged**